data/brands/amenity/bicycle_rental.json

```
@@ -1060,16 +1060,17 @@
       }
     },
     {
-      "displayName": "Citybike Wien",
-      "id": "citybikewien-cda769",
+      "displayName": "WienMobil Rad",
       "locationSet": {"include": ["at"]},
```

```suggestion
      "locationSet": {"include": ["at"]},
      "matchNames": ["Citybike Wien"],
```
If this is a name change modify you'd better add old name as a matchName.

---

They are not the same bicycle sharing system. Rather, the first one (Citybike Wien) expired its contract with the city council, which selected a different company (Nextbike) to run a new bicycle sharing system (WienMobil Rad) with new stations and bicycles.

---

WienMobil Rad started operations on the 1st April 2022, just one day after Citybike Wien stopped operating.